src/components/MapView/index.tsx

```diff
@@ -419,6 +419,18 @@ function MapView({ classes }: MapViewProps) {
     setFirstSymbolId(layers?.find(layer => layer.type === 'symbol')?.id);
     dispatch(setMap(() => map));
     map.jumpTo({ center: [longitude, latitude], zoom });
+    const maxBounds = get(appConfig.map, 'maxBounds');
+    const minZoom = get(appConfig.map, 'minZoom');
+    const maxZoom = get(appConfig.map, 'maxZoom');
+    if (maxBounds) {
+      map.setMaxBounds(maxBounds);
```

Comment:
what happens if you do setMaxBounds(undefined)?
could we do something like `map.setMaxBounds(maxBounds || undefined)`?